TalkAloud/TalkAloud/Model/CoreDataManager.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CoreDataManager.swift
+//  TalkAloud
+//
+//  Created by Justin Bengtson on 6/25/20.
+//  Copyright © 2020 Justin Bengtson. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class CoreDataManager: NSManagedObject {
+    
+    static let sharedInstance = CoreDataManager()
+    
+    private var audioRecording: AudioRecording?
+    private var audioRecordings = [AudioRecording]()
+    
+    func saveToCoreDataObject() -> AudioRecording? {
```

Comment:
FInd a better name because you are not saving to core data, you are creating a new AudioRecording object.